// Fail on unexpected error and warn
require('./fail-on-errors');
